mono/utils/mono-context.h

```diff
@@ -272,7 +273,10 @@ MONO_DISABLE_WARNING(4324) // 'struct_name' : structure was padded due to __decl
 
 typedef struct {
 	host_mgreg_t gregs [AMD64_NREG];
-#if defined(MONO_HAVE_SIMD_REG)
+#if defined(MONO_HAVE_SIMD_REG_AVX)
+	// Lower AMD64_XMM_NREG fregs holds lower 128 bit YMM. Upper AMD64_XMM_NREG fregs holds upper 128-bit YMM.
```

Comment:
Same

Reply:
I have grabbed mono-context.h from master and applied just the s390x patch. The diff now looks like:
```
--- a/mono/utils/mono-context.h
+++ b/mono/utils/mono-context.h
@@ -33,7 +33,7 @@ typedef struct __darwin_xmm_reg MonoContextSimdReg;
 typedef struct _libc_xmmreg MonoContextSimdReg;
 #elif defined(HOST_WIN32)
 #define MONO_HAVE_SIMD_REG
-#define MONO_HAVE_SIMD_REG_AVX
+//#define MONO_HAVE_SIMD_REG_AVX
 #include <emmintrin.h>
 typedef __m128d MonoContextSimdReg;
 #elif defined(HOST_ANDROID)
@@ -914,6 +914,7 @@ typedef struct ucontext MonoContext;
 #define MONO_CONTEXT_SET_BP(ctx,bp)                                    \
        do {                                                            \
                (ctx)->uc_mcontext.gregs[15] = (unsigned long)bp;       \
+               (ctx)->uc_stack.ss_sp        = (void*)bp;               \
        } while (0)

 #define MONO_CONTEXT_GET_IP(ctx) (gpointer) (ctx)->uc_mcontext.psw.addr
```
The diff is against my previous PR so will this pass muster?

Reply:
Just make sure that this PR contains only the changes necessary for s390x, not any additional changes that happened in the meantime in dotnet/runtime(lab).